locales-src/onboarding.ftl

```diff
@@ -31,12 +31,20 @@ onboarding-join-form-email =
 onboarding-join-form-email-error = Valid email required
 onboarding-join-form-legal = By proceeding, you agree to the <a data-l10n-name="terms">Terms of Service</a> and <a data-l10n-name="privacy">Privacy Notice</a>.
 onboarding-join-form-continue = Continue
+onboarding-join-form-signin-label = Already have an account?
```

Comment:
Can this be a single string?

```
onboarding-join-form-signin-label = Already have an account? <a data-l10n-name="sign-in">Sign In</a>
```

Reply:
The link was implemented as a button as it was reusing the other form input data as part of the "href" destination. Looks like fluent-dom 0.2.0 has limitations on adding event listeners: https://github.com/projectfluent/fluent.js/wiki/DOM-Overlays#limitations

I would suggest keeping the original separate strings to keep the code simpler.

Reply:
Updated patch to keep two strings separate for now. Thanks